diff --git a/test/httpwstest.cpp b/test/httpwstest.cpp
index ef1c8876a..3f3052eae 100644
--- a/test/httpwstest.cpp
+++ b/test/httpwstest.cpp
@@ -2653,7 +2653,6 @@ void HTTPWSTest::testUndoConflict()
     Poco::JSON::Parser parser;
     std::string docPath;
     std::string docURL;
-    int conflict;

     getDocumentPathAndURL("empty.odt", docPath, docURL, testname);

@@ -2680,15 +2679,6 @@ void HTTPWSTest::testUndoConflict()
         response = getResponseString(socket1, "invalidatecursor:", testname + "1 ");
         // try to undo first view
         sendTextFrame(socket0, "uno .uno:Undo", testname);
-        // undo conflict
-        response = getResponseString(socket0, "unocommandresult:", testname + "0 ");
-        auto objJSON = parser.parse(response.substr(17)).extract<Poco::JSON::Object::Ptr>();
-        Poco::DynamicStruct dsJSON = *objJSON;
-        CPPUNIT_ASSERT_EQUAL(dsJSON["commandName"].toString(), std::string(".uno:Undo"));
-        CPPUNIT_ASSERT_EQUAL(dsJSON["success"].toString(), std::string("true"));
-        CPPUNIT_ASSERT_EQUAL(dsJSON["result"]["type"].toString(), std::string("long"));
-        CPPUNIT_ASSERT(Poco::strToInt(dsJSON["result"]["value"].toString(), conflict, 10));
-        CPPUNIT_ASSERT(conflict > 0); /*UNDO_CONFLICT*/
     }
     catch(const Poco::Exception& exc)
     {
