From: Jochen Sprickerhof <git@jochen.sprickerhof.de>
Date: Tue, 1 Nov 2016 22:35:07 +0100
Subject: Use null as device file as console might not be there

---
 Foundation/testsuite/src/FileTest.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Foundation/testsuite/src/FileTest.cpp b/Foundation/testsuite/src/FileTest.cpp
index 2f63599..ea47484 100644
--- a/Foundation/testsuite/src/FileTest.cpp
+++ b/Foundation/testsuite/src/FileTest.cpp
@@ -231,7 +231,7 @@ void FileTest::testFileAttributes3()
 #if POCO_OS==POCO_OS_CYGWIN
 	File f("/dev/tty");
 #else
- 	File f("/dev/console");
+ 	File f("/dev/null");
 #endif
 #elif defined(POCO_OS_FAMILY_WINDOWS) && !defined(_WIN32_WCE)
 	File f("CON");
